core/controllers/admin_test.py

```diff
@@ -3854,38 +3854,55 @@ def test_admin_should_be_able_to_update_config(self) -> None:
 
         voiceover_services.update_admin_config_for_voiceover_autogeneration(
             False)
-        self.assertFalse(
-            voiceover_services.
-            is_voiceover_autogeneration_using_cloud_service_enabled()
+
+        voiceover_autogeneration_policy = (
+            voiceover_services.get_voiceover_autogeneration_policy())
+        autogenerated_voiceovers_are_enabled = (
```

Comment:
Don't have logic in tests. Try to find a way to test this without using if/else.

Ditto below and in all other test files.